crates/bevy_pbr/src/meshlet/gpu_scene.rs

```diff
@@ -0,0 +1,763 @@
+    view_uniforms: Res<ViewUniforms>,
+    render_device: Res<RenderDevice>,
+    mut commands: Commands,
+) {
+    let Some(view_uniforms) = view_uniforms.uniforms.binding() else {
+        return;
+    };
+
+    for (view_entity, view_resources, view_depth_texture) in &views {
+        let entries = BindGroupEntries::sequential((
+            gpu_scene.meshlets.binding(),
+            gpu_scene.instance_uniforms.binding().unwrap(),
+            gpu_scene.thread_instance_ids.binding().unwrap(),
+            gpu_scene.thread_meshlet_ids.binding().unwrap(),
+            gpu_scene.previous_thread_ids.binding().unwrap(),
+            view_resources.previous_occlusion_buffer.as_entire_binding(),
+            view_resources.occlusion_buffer.as_entire_binding(),
+            gpu_scene.meshlet_bounding_spheres.binding(),
+            view_resources
+                .visibility_buffer_draw_command_buffer_first
+                .as_entire_binding(),
+            view_resources
+                .visibility_buffer_draw_index_buffer
+                .as_entire_binding(),
+            view_uniforms.clone(),
+        ));
+        let culling_first = render_device.create_bind_group(
+            "meshlet_culling_first_bind_group",
+            &gpu_scene.culling_first_bind_group_layout,
+            &entries,
+        );
+
+        let entries = BindGroupEntries::sequential((
+            gpu_scene.meshlets.binding(),
+            gpu_scene.instance_uniforms.binding().unwrap(),
+            gpu_scene.thread_instance_ids.binding().unwrap(),
+            gpu_scene.thread_meshlet_ids.binding().unwrap(),
+            gpu_scene.previous_thread_ids.binding().unwrap(),
+            view_resources.previous_occlusion_buffer.as_entire_binding(),
+            view_resources.occlusion_buffer.as_entire_binding(),
+            gpu_scene.meshlet_bounding_spheres.binding(),
+            view_resources
+                .visibility_buffer_draw_command_buffer_second
+                .as_entire_binding(),
+            view_resources
+                .visibility_buffer_draw_index_buffer
+                .as_entire_binding(),
+            view_uniforms.clone(),
+            &view_resources.depth_pyramid.default_view,
+            &gpu_scene.depth_pyramid_sampler,
+        ));
+        let culling_second = render_device.create_bind_group(
+            "meshlet_culling_second_bind_group",
+            &gpu_scene.culling_second_bind_group_layout,
+            &entries,
+        );
+
+        let downsample_depth = [0, 1, 2, 3, 4, 5].map(|i| {
+            render_device.create_bind_group(
+                "meshlet_downsample_depth_bind_group",
+                &gpu_scene.downsample_depth_bind_group_layout,
+                &BindGroupEntries::sequential((
+                    if i == 0 {
+                        view_depth_texture.view()
+                    } else {
+                        &view_resources.depth_pyramid_mips[i - 1]
```

Comment:
It's under the "future work" section. I won't be doing it in this PR because it's a lot more complicated to get working for every possible texture size :P.

> Single pass compute shader depth pyramid generation